virtcontainers/kata_agent.go

```diff
@@ -1305,6 +1299,8 @@ func (k *kataAgent) createContainer(sandbox *Sandbox, c *Container) (p *Process,
 		return nil, err
 	}
 
+	c.sandbox.config.HypervisorConfig.ProcessLabel = grpcSpec.Process.SelinuxLabel
```

Comment:
this will not work, `k.createContainer` is called after `startSandbox`

Reply:
Where should I call this?  I need to get the label out of the OCI spec before the VM is launched.

Reply:
Figured it out now setting the ProcessLabel inside of startSandbox.